### PR TITLE
support for pad titles, limit number of entries shown, sort by last change and reverse order

### PR DESCRIPTION
**Pull request by rittmeier**
I would like to display the last 10 pads changed on the index page.

Therefore, I added several optional settings:
* limit number of entries
* influence sort order, so that the list is sorted by date of last change of the pad
* reverse the order

In addition support for ep_set_title_on_pad is added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,42 @@ A small script for [etherpad-lite](https://github.com/ether/etherpad-lite) which
...
+}
+```
+
+Default is to use the order in which the API returns the pads ("default").
+
+### Reverse order
+
+```json
+"ep_small_list": {
+  "order": "reverse"
```
[the remark this answers is not in the document]
Comment (rittmeier) [reply README.md]: The order setting is now renamed to sortorder. Also updates the README regarding the available options. Other sorting orders are currently not implemented. I`m not sure which other sorting orders would be helpfull.

File: README.md [hunk 2]
```diff
@@ -8,3 +8,42 @@ A small script for [etherpad-lite](https://github.com/ether/etherpad-lite) which
...
+}
+```
+
+By default the number of entries is not limited ("limit": 0).
+
+### Sort by last change
+
+```json
+"ep_small_list": {
+  "sortmode": "lastchange"
```
[the remark this answers is not in the document]
Comment (rittmeier) [reply README.md:30]: Added.

File: index.js [hunk 3]
```diff
@@ -1,33 +1,105 @@
...
-            res.send("<div> Error:" + err + "</div>");
-        })
-      }
     });
+  });
 };
 
-function createList(data){
-  r = "<ul>";
+async function createList(data){
```
[the remark this answers is not in the document]
Comment (rittmeier) [reply index.js:53]: Now added. :-)

File: index.js [hunk 4]
```diff
@@ -1,33 +1,105 @@
...
+      createList(pads).then(function(html){
+        res.send(html);
       });
-      if(pads){
-        pads.then(function(data){
-          res.send(createList(data));
-        }).catch(function(err){
-            res.send("<div> Error:" + err + "</div>");
-        })
-      }
```
[the remark this answers is not in the document]
Comment (rittmeier) [reply index.js:20]: 🙈

[the remark this answers is not in the document]
Comment (rittmeier) [reply index.js:20]: added and tested.

**Comment by spruce** [comment]
Thanks ❤️

**Comment by rittmeier** [comment]
Would you mind releasing a new version to npm? :-)

**Comment by spruce** [comment]
will do this evening.

**Comment by spruce** [comment]
done.

**Pull request merged**